stories/modules-tagmanager-components.stories.js

```diff
@@ -0,0 +1,147 @@
+			registry.dispatch( STORE_NAME ).receiveGetAccounts( fixtures.accounts );
+			registry.dispatch( STORE_NAME ).receiveGetExistingTag( null );
+			registry.dispatch( STORE_NAME ).receiveGetSettings( {} );
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<AccountSelect />
+			</WithTestRegistry>
+		);
+	} )
+	.add( 'WebContainerSelect', () => {
+		const setupRegistry = ( registry ) => {
+			const accountID = fixtures.getContainers.all[ 0 ].accountId;
+			registry.dispatch( STORE_NAME ).receiveGetSettings( {} );
+			registry.dispatch( STORE_NAME ).receiveGetAccounts( [] );
+			registry.dispatch( STORE_NAME ).setAccountID( accountID );
+			registry.dispatch( STORE_NAME ).receiveGetContainers( fixtures.getContainers.all, { accountID } );
+			registry.dispatch( STORE_NAME ).receiveGetExistingTag( null );
+			registry.dispatch( CORE_SITE ).receiveSiteInfo( {} );
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<WebContainerSelect />
+			</WithTestRegistry>
+		);
+	} )
+	.add( 'AMPContainerSelect', () => {
+		const setupRegistry = ( registry ) => {
+			const accountID = fixtures.getContainers.all[ 0 ].accountId;
+			registry.dispatch( STORE_NAME ).receiveGetSettings( {} );
+			registry.dispatch( STORE_NAME ).receiveGetAccounts( [] );
+			registry.dispatch( STORE_NAME ).setAccountID( accountID );
+			registry.dispatch( STORE_NAME ).receiveGetContainers( fixtures.getContainers.all, { accountID } );
+			registry.dispatch( STORE_NAME ).receiveGetExistingTag( null );
+			registry.dispatch( CORE_SITE ).receiveSiteInfo( { ampMode: AMP_MODE_PRIMARY } );
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<AMPContainerSelect />
+			</WithTestRegistry>
+		);
+	} )
+	.add( 'UseSnippetSwitch', () => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( STORE_NAME ).setSettings( {} );
+			registry.dispatch( STORE_NAME ).setUseSnippet( true );
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<UseSnippetSwitch />
+			</WithTestRegistry>
+		);
+	} )
+	.add( 'ExistingTagNotice', () => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( STORE_NAME ).receiveGetExistingTag( 'GTM-S1T3K1T' );
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<ExistingTagNotice />
+			</WithTestRegistry>
+		);
+	} )
+	.add( 'ExistingTagError', () => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( STORE_NAME ).receiveGetExistingTag( 'GTM-S1T3K1T' );
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<ExistingTagError />
+			</WithTestRegistry>
+		);
+	} )
+	.add( 'ErrorNotice', () => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( STORE_NAME ).receiveError( {
+				code: 'KavHBokm704',
+				message: 'They taste like burning!',
+			} );
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<ErrorNotice />
+			</WithTestRegistry>
+		);
+	} )
```

Comment:
I think we can omit this component from here, we don't test it elsewhere and it's really just duplicated across all modules right now. I'll open an issue for implementing a single unified variant of it.